specification/developerhub/resource-manager/Microsoft.DevHub/preview/2022-10-11-preview/workflow.json

```diff
@@ -0,0 +1,1154 @@
+          {
+            "value": "enabled",
+            "description": "Manifests will be generated"
+          },
+          {
+            "value": "disabled",
+            "description": "Manifests will not be generated"
+          }
+        ]
+      },
+      "description": "The mode of generation to be used for generating Manifest."
+    },
+    "DockerfileGenerationMode": {
+      "type": "string",
+      "enum": [
+        "enabled",
+        "disabled"
+      ],
+      "x-ms-enum": {
+        "name": "DockerfileGenerationMode",
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "enabled",
+            "description": "Dockerfiles will be generated"
+          },
+          {
+            "value": "disabled",
+            "description": "Dockerfiles will not be generated"
+          }
+        ]
+      },
+      "description": "The mode of generation to be used for generating Dockerfiles."
+    },
+    "GenerationManifestType": {
+      "type": "string",
+      "enum": [
+        "helm",
+        "kube"
+      ],
+      "x-ms-enum": {
+        "name": "GenerationManifestType",
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "helm",
+            "description": "Helm manifests"
+          },
+          {
+            "value": "kube",
+            "description": "Kubernetes manifests"
+          }
+        ]
+      },
+      "description": "Determines the type of manifests to be generated."
+    },
+    "WorkflowRun": {
+      "type": "object",
+      "properties": {
+        "succeeded": {
+          "type": "boolean",
+          "readOnly": true,
+          "description": "Describes if the workflow run succeeded."
+        },
+        "workflowRunURL": {
+          "type": "string",
+          "readOnly": true,
+          "description": "URL to the run of the workflow."
+        },
+        "lastRunAt": {
+          "type": "string",
+          "readOnly": true,
+          "format": "date-time",
+          "description": "The timestamp of the last workflow run."
+        }
+      }
+    },
+    "ACR": {
+      "type": "object",
+      "description": "Information on the azure container registry",
+      "properties": {
+        "acrSubscriptionId": {
+          "type": "string",
+          "description": "ACR subscription id"
+        },
+        "acrResourceGroup": {
+          "type": "string",
+          "description": "ACR resource group"
+        },
+        "acrRegistryName": {
+          "type": "string",
+          "description": "ACR registry"
+        },
+        "acrRepositoryName": {
+          "type": "string",
+          "description": "ACR repository"
+        }
+      }
+    },
+    "GeneratePreviewArtifactsResponse": {
+      "type": "object",
+      "description": "Dockerfile and manifest artifacts generated as a preview are returned as a map<path string,content string>",
+      "properties": {
+        "path/to/file1.withExtension": {
```

Comment:
Dont think this is valid properties, I think it should be like:

"type": "object",
"additionalProperties": {
    "type": "string"
},

Reply:
thanks!